boards/modalai/voxl2/target/voxl-px4-hitl

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+CONFIG_FILE="/etc/modalai/voxl-px4-hitl.conf"
+
+DAEMON_MODE=DISABLE
+
+# Try to source configuration variables from a file.
+# This will override anything already set in the environment.
+if [ -f "$CONFIG_FILE" ]; then
+    echo "[INFO] Reading from $CONFIG_FILE"
+    source $CONFIG_FILE
+else
+    echo "[INFO] $CONFIG_FILE not found, using defaults"
+fi
+
+# Make sure that the SLPI DSP test signature is there otherwise px4 cannot run
+# on the DSP
+if /bin/ls /usr/lib/rfsa/adsp/testsig-*.so &> /dev/null; then
+    /bin/echo "Found DSP signature file"
+else
+    /bin/echo "[WARNING] Could not find DSP signature file"
+    # Look for the DSP signature generation script
+    if [ -f /share/modalai/qrb5165-slpi-test-sig/generate-test-sig.sh ]; then
+        /bin/echo "[INFO] Attempting to generate the DSP signature file"
+        # Automatically generate the test signature so that px4 can run on SLPI DSP
+        /share/modalai/qrb5165-slpi-test-sig/generate-test-sig.sh
+    else
+        /bin/echo "[ERROR] Could not find the DSP signature file generation script"
+        exit 0
+    fi
+fi
+
+print_usage() {
+    echo -e "\nUsage: voxl-px4-hitl"
+    echo "                [-c delete configuration file and exit]"
+    echo "                [-d start px4 in daemon mode]"
+    echo "                [-h (show help)]"
+
+    exit 1
+}
+
+while getopts "cdhz" flag
+do
+    case "${flag}" in
+        c) 
+            echo "[INFO] Wiping old config file"
+            if [ -f "$CONFIG_FILE" ]; then
+                rm -rf ${CONFIG_FILE}
+            fi
+            exit 0
+            ;;
+        d) 
+            echo "[INFO] Enabling daemon mode"
+            DAEMON_MODE=ENABLE
+            ;;
+        h) 
+            print_usage
+            ;;
+        *) 
+            print_usage
+            ;;
+    esac
+done
+
+if [ $DAEMON_MODE == "DISABLE" ]; then
+    DAEMON=" "
+else
+    echo "[INFO] Daemon mode enabled"
+    DAEMON="-d"
+fi
+
+if [ ! -f /data/px4/param/hitl_parameters ]; then
+   echo "[INFO] Setting default parameters for PX4 on voxl"
+   px4 $DAEMON -s /etc/modalai/voxl-px4-hitl-set-default-parameters.config
```

Comment:
I don't think this will work? That parameter setting script will try to shutdown at the end which may not work reliably?